dataprofiler/profilers/__init__.py

```diff
@@ -67,6 +68,7 @@
 json_decoder._options = {
     BooleanOption.__name__: BooleanOption,
     HistogramOption.__name__: HistogramOption,
```

Comment:
what if instead of `HistogramOption.__name__: HistogramOption,` we just did:
`"HistogramOption": HistogramAndQuantilesOption,`

Could that remove a bunch of duplication

Reply:
I think the deprecation would come at loading time in json_decoder